test/broadcasttests.jl

```diff
@@ -150,4 +150,9 @@ import Base: broadcasted
         @test arguments(Vt) == (transpose(B.args[1][1:3,1:2]), permutedims(B.args[2][1:3]))
         @test BroadcastArray(Vc) == BroadcastArray(Vt) == Vc == (Array(B)')[1:2,1:3]      
     end
+
+    @testset "copy to TrackedArray" begin
+        a = LazyArray(broadcasted(+, param(rand(3, 3)), 1))
+        @test copy(a) isa TrackedArray
```

Comment:
Add an `@inferred` to make sure your routine is type stable.

Reply:
Done